org-code-javabuilder/validation/src/main/java/org/code/validation/support/ExceptionKey.java

```diff
@@ -0,0 +1,8 @@
+package org.code.validation.support;
+
+public enum ExceptionKey {
+  // The user's main method could not be run
+  ERROR_RUNNING_MAIN,
+  // The user has no main method and we tried to run main
+  NO_MAIN_METHOD
```

Comment:
Is this the same as the exception key for a missing main method when running code? How do we differentiate between the two?

Reply:
right now we differentiate because `ValidationRuntimeException` is what gets sent this enum value, and it translates it into a separate error message. But it might make sense to make it distinct so that we can add errors on the Java Lab side when we add translation here.

Reply:
updated to a different enum value